docker-compose.yml

```diff
@@ -166,6 +167,9 @@ services:
       - ${TMP_ENV_CI:-.env.ci}
     volumes:
       - ./:/repo
+    environment:
+      - KRATOS_PUBLIC_API=http://kratos:4433
```

Comment:
While I like having these settings here (and I believe in theory we could now get rid of `.env.ci` I think for the time being I would add these values to `.env.ci` to have all the environment variables colocated.